tests/transforms/test_pauli_based_computation.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2018-2023 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for to_ppr, commute_ppr, merge_ppr_ppm, ppm_compilation, ppr_to_ppm, and reduce_t_depth (not implemented with tape)."""
+
+
+import pytest
+
+import pennylane as qml
+from pennylane.transforms.decompositions import (
+    commute_ppr,
+    merge_ppr_ppm,
+    ppm_compilation,
+    ppr_to_ppm,
+    reduce_t_depth,
+    to_ppr,
+)
+
+PBC_TRANSFORMS = [to_ppr, commute_ppr, merge_ppr_ppm, ppm_compilation, ppr_to_ppm, reduce_t_depth]
+PASS_NAMES = [
+    "to-ppr",
+    "commute_ppr",
+    "merge_ppr_ppm",
+    "ppm-compilation",
+    "ppr_to_ppm",
+    "reduce-t-depth",
+]
+PBC_TRANSFORM_DATA = list(zip(PBC_TRANSFORMS, PASS_NAMES))
+
+
+@pytest.mark.parametrize("pbc_transform, pass_name", PBC_TRANSFORM_DATA)
+class TestPBCTransforms:
```

Comment:
I'd prefer if we keep the test names a bit more readable.

```suggestion
class TestPauliBasedComputationTransforms:
```